ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -53,6 +53,18 @@ public static int findTargetTagForTouch(
     return targetTag;
   }
 
+  public static View findTargetViewForTouch(
```

Comment:
This is a simpler version of `findTargetTagForTouch`. Could share code instead of copy&pasting.

Missing documentation.